parsers/US_PJM.py

```diff
@@ -4,17 +4,21 @@
 
 import json
 import re
-from datetime import datetime
+from datetime import datetime, timedelta
 from logging import Logger, getLogger
-from typing import List, Optional, Union
+from typing import Dict, List, Optional, Union
 
 import arrow
 import demjson3 as demjson
+import pandas as pd
+import pytz
 from bs4 import BeautifulSoup
 from dateutil import parser, tz
 from requests import Session, get
```

Comment:
```suggestion
from requests import Session, Response
```
Let's just the `Session.get()` function directly instead of importing get (unless there is a reason?).